### ISSUE TITLE
Update color_blue

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#143]
As discussed on https://github.com/Yoast/my-yoast/issues/362
the color_blue should be changed from `#0075b3` to `#0066cd` to be more accessible, even against backgrounds like `#f1f1f1`.
